lib/node_modules/@stdlib/stats/base/nanvariancech/test/test.nanvariancech.js

```diff
@@ -230,15 +352,36 @@ tape( 'the function supports a `stride` parameter', function test( t ) {
 		NaN
 	];
 
-	N = floor( x.length / 2 );
-	v = nanvariancech( N, 1, x, 2 );
+	v = nanvariancech( 4, 1, x, 2 );
+
+	t.strictEqual( v, 6.25, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the function supports a `stride` parameter (accessors)', function test( t ) {
+	var x;
+	var v;
+
+	x = [
+		1.0,  // 0
+		2.0,
+		2.0,  // 1
+		-7.0,
+		-2.0, // 2
+		3.0,
+		4.0,  // 3
+		2.0,
+		NaN,  // 4
+		NaN
+	];
+
+	v = nanvariancech( 4, 1, toAccessorArray( x ), 2 );
```

Comment:
```suggestion
	v = nanvariancech( 5, 1, toAccessorArray( x ), 2 );
```
notice in the comments that the indices which should be traversed are mentioned, kindly follow that

Reply:
same for all other instances.